documentation/release_notes.rst

```diff
@@ -8,6 +8,21 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.1.1
+===============
+New Features
+------------
+- Improved ``sort`` algorithm performance for the arithmetic data types with ``std::less`` or ``std::greater`` comparison operator and DPC++ policy.
+- AOT compilation for CPU devices is now possible with the use of ``ONEDPL_USE_AOT_COMPILATION`` cmake option.
```

Comment:
As far as I remember, it was supposed to only affect the tests, and be named appropriately.
If we plan to change the name, maybe it makes sense to postpone this note until we make that change. If we do not plan to change the name, at the very least the note should clarify the applicability scope.

Reply:
Currently, this macro affects external targets as well. However, it might be better to use it only for the test only (prons: it is not related to oneDPL; cons: still, it provides some shortcut for external targets) and thus do not document it in release notes.

Reply:
I suggest omitting the notes about `ONEDPL_USE_AOT_COMPILATION` cmake option.

Reply:
Omitted the notes about ONEDPL_USE_AOT_COMPILATION cmake option as was suggested.